quivr/tables.py

```diff
@@ -594,8 +594,7 @@ def select(self, column_name: str, value: Any) -> Self:
         :param column_name: The name of the column to select on.
```

Comment:
Please add documentation that the column_name can be dot-delimited in the manner used by column().